packages/esm-patient-search-app/src/patient-search-page/patient-banner/banner/patient-banner.scss

```diff
@@ -9,6 +10,36 @@
   grid-template-columns: 1fr auto;
 }
 
+.activePatientContainer {
+  background-color: $ui-02;
+}
+
+.deceasedPatientContainer {
+  background-color: colors.$gray-80;
+
+  .patientName {
+    color: $ui-01;
+  }
+
+  .demographics,
+  .identifiers {
+    color: $ui-02;
+  }
+
+  .actionsButtonText {
```

Comment:
Do we need to apply this color, i.e., is it any different from the standard color already there?

Reply:
We do need to apply that colour without it the identifier and demographic attributes are hardly seen.